### ISSUE TITLE
piercing shots, righteous vengeance, unholy blight vs absorb effects. 

### ISSUE THREAD

**Issue by dathremar** [SunwellTracker/issues#946]
Decription: 
Talents and effects mentioned in the headline bug when part of the damage of the basic spell (judgement,chimera shot, deathcoil etc) of which the damage is recalculated gets absorbed

How it works: 
It works the same with all 3, so im gonna give piercing shots as an example:
Piercing shots proc from chimera shot, aimed shot and steady shot crits,
Lets say we hava a hunter facing a priest with 5k shield.
Hunter crits 4k with chimera shot - its entire dmg gets absorbed, piercing shot will do a total of 0 dmg (8x miss communicate) instead of 4000*0.3/8 with each tick
Hunter crits 6k with chimera shots - he deals 1000 damage+5k absorb damage, piercing shots will do 1000*0.3/8 damage with each tick, instead of 6000*0.3/8

When it comes to unholy blight the death coil doesnt have to crit, but if its whole damage is absorbed then unholy blight misses with each tick, and when part of its damage gets absorb UB has reduced damage by proportional amount.

How it should work: 
Absorbs should have no effect on damage calculation from aforementioned talents. The formula should be (dmg done+dmg absorbed)*(% modifier),  
e.q piercing shots shoud be (dmg done+dmg absorbed)*0.3

Source (you should point out proofs of your report, please give us some source): 

https://www.youtube.com/watch?v=rz33eYy553s&  - Rightous Vengeance - 10:03 - Paladin uses judgement on priest it deals 1354 damage (rest was absorbed), but righteous vengeance ticks for 200, which is way too high for 1354 crit.

https://www.youtube.com/watch?v=gi3j4VGSDHQ - piercing shots - 8:53 - hunter gets 4262 chimera shot crit on rog, rog gets shielded by his priest, hunter uses aimed shot on rog (we can see that piercing shots debuff gets refreshed after aimed shot - that means it was a fully absorbed crit), then rog's shield gets taken down and we can see that piercing shots tick for 269 (269*8 is not 30% from 4262, it means that piercing shot damage was increased by that fully absorbed aimed shot crit   - 
269*8/0.3=7173  
7173-4264=2911 which equals to that absorbed aimed shot crit damage  - could actually be higher cus the bleed dmg might have been modified by resilence, altough its irrelevant in regards to bug described in this report, we know for certain that absorbs should not affect damage calculation of piercing shots)

https://db.darkwizard.pl/?spell=53382
https://db.darkwizard.pl/?spell=49194
https://db.darkwizard.pl/?spell=53238

**Comment by dathremar** [comment]
https://github.com/SunwellTracker/issues/issues/43

**Comment by knasen88** [comment]
dw, i'm sure they'll get to this after fixing some shitty quest in azshara or fixing the flavour text of some hellfire quest prior. /salt

**Comment by dathremar** [comment]
yeah i guess ur right, i reported this bug before angrathar was even released and still no fix ;f
